go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/plgd-dev/hub/v2
 
-go 1.22
+go 1.23
 
-toolchain go1.22.0
+toolchain go1.23.0
```

Comment:
could we leave it on 1.22 - just in go.mod

Reply:
OK, lets keep it at 1.22 for now